packages/dx-react-grid/docs/reference/table-edit-row.md

```diff
@@ -15,6 +15,7 @@ Name | Type | Default | Description
 -----|------|---------|------------
 rowHeight | number | | Specifies the edit row height
 editCellTemplate | (args: [EditCellArgs](#edit-cell-args)) => ReactElement | | A component that renders an editable cell
+editRowTemplate | (args: [TableRowArgs](table-view.md#table-row-args)) => ReactElement | | A component that renders an editable row
```

Comment:
Shouldn't we stress here and in other similar places that it renders row *container* not the whole row with content?